std/compress/stream.go

```diff
@@ -31,12 +31,21 @@ func (s *Stream) At(i int) int {
 	return s.D[i]
 }
 
-func NewStreamFromBytes(in []byte) Stream {
-	d := make([]int, len(in))
-	for i := range in {
-		d[i] = int(in[i])
+func panicIfErr(err error) {
+	if err != nil {
```

Comment:
I would rather panics be real panics; if they are actually reachable due to a malformed input or something like that, it's a potential (unlikely but still ^^) ddos vector